config/initializers/rollbar.rb

```diff
@@ -10,8 +10,8 @@
 
   config.environment = ENV['ROLLBAR_ENV']
 
-  # Here we'll disable in 'test':
-  config.enabled = false if Rails.env.test?
+  # Here we'll disable in 'test' and 'development':
+  config.enabled = false if Rails.env.test? or Rails.env.development?
```

Comment:
@trystant `or` [has super weird operator precedence](https://stackoverflow.com/questions/2083112/difference-between-or-and-in-ruby), so I'd generally go with `||` in situations like this. That being said, [I tried a similar change](https://github.com/EBWiki/EBWiki/pull/3883) before I saw this PR, but it sounds like we might want to discuss what approach to take with this.

Reply:
@mockdeep Yeah, `or` is lower on the [Ruby operator precedence totem pole](https://stackoverflow.com/questions/21060234/ruby-operator-precedence-table). I'll change that up.